RELEASE.md

```diff
@@ -0,0 +1,15 @@
+Release type: patch
+
+This release fixes the upload of nested file lists. Among other use cases, having an
+input type like shown below is now working properly.
+
+```python
+import typing
+import strawberry
+from strawberry.file_uploads import Upload
+
+
+@strawberry.input
+class FolderInput:
+    files: typing.List[Upload]
```

Comment:
good catch! I haven't used file uploads much, so I didn't consider this use case 😊

Reply:
I also came across it only by chance while extending my python graphql clients tests to run against a Strawberry app :)